### PR TITLE
feat: Provider & ViewModel & enum & Databinding created

### LINKED ISSUE #9: [Weather:UI] UI & Provider & Navigator 작성
## 목적
UI & Provider & Navigator 작성

## 상세 사항
UI
-wireframe 

Dependency
-Provider 주입

Navigator
-route 및 Hero 생성

### PR DESCRIPTION
**Pull request by Gunbam27**
feat: Provider & ViewModel & enum & Databinding created

### CODE REVIEW

**Comment by jungeun272** [comment]
... 왜이렇게 많이하셨어요

**Comment by frihett** [comment]
wonderful

**Comment by chojja7188** [comment]
멋져요~!  👍👍

**Comment by sungchanjung** [comment]
너무 고생하셨어요 ~

File: assets/images/Halla.png
Comment (chojja7188) [inline assets/images/Halla.png:1]: 산이 참 멋지네요

**Pull request merged**